webapp/components/test-search.js

```diff
@@ -215,10 +224,12 @@ const QUERY_SEMANTICS = QUERY_GRAMMAR.createSemantics().addOperation('eval', {
         return { lessThan: parseInt(n.eval()) + 1 };
       case "<":
         return { lessThan: n.eval() };
+      case ":":
+      case "=":
```

Comment:
This is something new in the syntax, right? Can you document what `=` does?

Reply:
`count=1(...)` is equivalent to `count:1(...)`. It seems inconsistent to support `count>1` but not `count=1`. This PR just simplifies where equality is consumed.